docs/reference/error-codes.mdx

```diff
@@ -270,7 +270,10 @@ it is called at app initialization, not in response to each request:
 + app.use(rateLimitFactory()); // Works
 ```
 
-Set `validate: {creationStack: false}` in the options to disable the check.
+Set `validate: {creationStack: false}` in the options to disable the check, only
+if you are sure that the rate limiter is working as intended, and is not facing
+performance problems (that may be caused by repeated initialization of an
+external data store).
```

Comment:
Perhaps this should be it's own paragraph. Something like this:

```suggestion
In certain advanced use cases, such as serving multiple websites from a single app, it may be necessary to dynamically create a rate limiter in response to a request. In this case the validation check may be disabled, but care should be taken to ensure everything works as expected. Consider caching the limiter after it's first usage to avoid repeating the initialization cost and any background work the store may require.

Set `validate: {creationStack: false}` in the options to disable the check.
```